Practice1/.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+indent_size = 4
+indent-style = tab
+end_of_line = crlf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 80
```

Comment:
Please check the EOF warning here and all the same cases below.

Reply:
Fixed